tests/IsoCodes/Tests/CreditCardTest.php

```diff
@@ -30,7 +30,6 @@ public function getValidCreditCards()
             array('4903010000000009'),  //Switch
             array('4111111111111111'),  //Visa
             array('6304100000000008'),  //Laser
-            array(6304100000000008),    //Laser
```

Comment:
yes obviously, but #75 is not about valid/invalid test dataset, but rather about valid/invalid 32bits OS. So `array(6304100000000008)` remains a 100% valid test.